util/src/main/java/io/kubernetes/client/util/ModelMapper.java

```diff
@@ -53,12 +53,13 @@
   // Model's api-version midfix to kubernetes api-version
   private static List<String> preBuiltApiVersions = new ArrayList<>();
 
-  // TODO(yue9944882): make the map bi-directional
   private static Map<GroupVersionKind, Class<?>> classesByGVK = new ConcurrentHashMap<>();
-
-  // TODO(yue9944882): make the map bi-directional
   private static Map<GroupVersionResource, Class<?>> classesByGVR = new ConcurrentHashMap<>();
 
+  private static Map<Class<?>, GroupVersionKind> gvkByClasses = new ConcurrentHashMap<>();
```

Comment:
I think we should encapsulate these two maps into a single bi-directional map. Ok for this to be a TODO.